Client/public/locales/tr.json

```
@@ -0,0 +1,103 @@
+{
+  "dontHaveAccount": "Hesabın yok mu",
+  "email": "E-posta",
+  "forgotPassword": "Şifremi Unuttum",
+  "password": "şifre",
+  "signUp": "Kayıt ol",
+  "submit": "Gönder",
+  "title": "Başlık",
+  "continue": "Devam et",
+  "enterEmail": "E-posta adresinizi girin",
+  "authLoginTitle": "Giriş Yap",
```

Biome (1.9.4)

[error] The key authLoginTitle was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)